ror-tools-core/src/main/scala/tech/beshu/ror/tools/core/patches/internal/filePatchers/JarManifestModifier.scala

```diff
@@ -40,11 +40,11 @@ object JarManifestModifier {
         ex => throw IllegalStateException(s"Could not copy content of jar file ${file.name} because of [${ex.getMessage}]", ex),
         (_: Unit) => ()
       )
-      tempJarFile.moveTo(file)(File.CopyOptions(overwrite = true))
     }.fold(
       ex => throw IllegalStateException(s"Could not add ROR version to jar file ${file.name} because of [${ex.getMessage}]", ex),
-      (_: File) => ()
+      (_: Unit) => ()
     )
+    tempJarFile.moveTo(file)(File.CopyOptions(overwrite = true))
```

Comment:
On Windows it was not possible to modify `file` inside the Using block, because it was opened as `jarFile`. On Linux it worked.